### PR TITLE
CIP-0013 | Acceptance Criteria: add more implementations

### PR DESCRIPTION
**Pull request by MadOrkestra**
Updated Acceptance Criteria with current state of wallet support

---

([rendered update](https://github.com/MadOrkestra/CIPs/blob/master/CIP-0013/README.md#path-to-active))

### CODE REVIEW

File: CIP-0013/README.md [hunk 1]
```diff
@@ -185,11 +185,15 @@ For a CIP based on this principle, see [CIP-0017](https://github.com/cardano-fou
 
 - [x] There exist one or more wallets supporting Payment URIs.
   - [x] Yoroi
+  - [x] beginWallet
```
Comment (rphair) [inline CIP-0013/README.md:188]:
```suggestion
  - [x] Begin Wallet
```
On the web site it's spelled consistently either `Begin Wallet` or simply `Begin`.

File: CIP-0013/README.md [hunk 2]
```diff
@@ -185,11 +185,15 @@ For a CIP based on this principle, see [CIP-0017](https://github.com/cardano-fou
...
 - [x] There exist one or more wallets supporting Stake Pool URIs.
   - [ ] TBD
 - [x] There exist other CIPs or drafts defining additional URI protocols.
-- [ ] There exist one or more wallets supporting additional URI protocols.
-  - [ ] TBD
+  - [x] [CIP-0099](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0099/README.md)  
+- [x] There exist one or more wallets supporting additional URI protocols.
+  - [x] Yoroi (CIP-0099)
+  - [x] beginWallet (CIP-0099)
```
Comment (rphair) [inline CIP-0013/README.md:195]:
```suggestion
  - [x] Begin Wallet (CIP-0099)
```
(continued)

**Review by rphair — COMMENTED** [review]
@MadOrkestra the format is OK and it does make sense to include CIP-0099 in the "additional protocols" & seems safe enough to include CIP-0099 supporters Yoroi and VESPR since they were the flagship implementors of CIP-0099.

@Crypto2099 how do we verify the inclusions of Begin Wallet?  I do recall their affirmative statements on Twitter regarding CIP-0013 & CPS-0016 but their web site is rather hollow & I don't see a lot of work on GitHub.  cc @Ryun1

**Comment by MadOrkestra** [comment]
@rphair Good points regarding Begin... I worked closely with them on this, so I can just reach out and request what we need? Tested CIP-13 on their most recent version on iOS.

**Comment by rphair** [comment]
yes @MadOrkestra you're welcome to present their statement here and/or ask them to comment in this thread directly.

**Comment by MadOrkestra** [comment]
I invited Francis from Begin to this thread, he'll hopefully join us and clear up all questions you might have.

**Comment by rphair** [comment]
@MadOrkestra FYI tagging this `Triage` puts it on the agenda for our next CIP meeting (https://hackmd.io/@cip-editors/96) so it might progress to `Confirmed` if you can get the Begin people to provide verification by then.

**Comment by MadOrkestra** [comment]
Noted! Believe me, I am trying :D

**Comment by francisluz** [comment]
@rphair do you need any extra detail?

@MadOrkestra Thanks to invite us and sorry for the delay.

**Review by rphair — APPROVED** [review]
@Ryun1 @Crypto2099 this is just a maintenance update for CIP-0013 so unless anyone expresses doubt here about the latest implementation there's point confirming it in a CIP meeting: so taking it off `Triage` & should be ready to confirm here.

**Comment by rphair** [comment]
> @francisluz: @rphair do you need any extra detail?

Thanks: we don't need any extra detail in the CIP itself but would help (though not a requirement for approval) to provide a link on your web site, documentation, and/or any developer channel indicating that you support CIP-0013 links.  If you have no such links, it will help if you create them.

For a long time we've been working to assure CIP-0013 support across more wallets for the lesser supported protocols, so more public support would create some pressure with the development community and the public.

Also such a statement would help us understand why wallet developers (and you) are choosing not to support the `stake:` links (yet): https://github.com/cardano-foundation/CIPs/tree/master/CIP-0013#stake-pool-uri-queries

**Comment by francisluz** [comment]
@rphair For the moment we don't have any extra documentation about the CIPs we support but we've it in our pipeline to create a comprehensive documentation about us.

Regarding Staking url was only dev time, we'll be adding it to our backlog too.

Appreciate

**Comment by rphair** [comment]
@Ryun1 FYI we missed tagging this as `Update` so it was showing up on the [Candidate list](https://github.com/cardano-foundation/CIPs/pulls?q=is%3Apr+is%3Aopen+label%3A%22State%3A+Confirmed%22%2C%22State%3A+Last+Check%22+draft%3Afalse+-label%3AUpdate%2CCorrection%2CTranslation%2C%22Bi-Weekly+Notes+%2F+Editorial+Housekeeping%22%2C%22CIP-0010%3A+new+registry+entry%22%2C%22CIP-0067%3A+new+label%22%2C%22CIP-0088%3A+new+extension%22+) 😅 (in case you see any others like it, e.g. on the new version of the front `README`, this is easily fixed by applying the `Update` tag etc.)

**Review by Ryun1 — APPROVED** [review]

**Pull request merged**